app/views/layouts/application.html.erb

```diff
@@ -43,7 +43,7 @@
     <%= govuk_skip_link %>
 
     <%= govuk_header(
-      classes: "app-header",
+      classes: current_user && current_user.support? ? "app-header app-header--orange" : "app-header",
```

Comment:
I wonder if these values (header class, phase banner attributes or just if logged in support user or not) should be derided somewhere in the application logic, rather than in the view?

Reply:
Also easier to test that way